test/parallel/test-tls-client-mindhsize.js

```diff
@@ -54,7 +54,7 @@ function test(size, err, next) {
       client.on('error', function(e) {
         nerror++;
         assert.strictEqual(e.message,
-                           'DH parameter size 1024 is less than 2048');
+                           'ERR_DH_PARAM_SIZE');
```

Comment:
Sorry for the incorrect observation above.
Now, things seems to be resolved. 
Thanks!